packages/ra-supabase-ui-materialui/src/AuthLayout.tsx

```diff
@@ -35,6 +35,8 @@ import { defaultTheme, Notification } from 'ra-ui-materialui';
  *             ...
  *        </Admin>
  *     );
+ *
+ * @deprecated Use the <Login> component from ra-ui-materialui instead
```

Comment:
What about the other authentication related pages such as `SetPasswordPage` and `ForgotPasswordPage`?